### ISSUE TITLE
Deleted Comfy UI and Hunyan to reinstall and fix it like others BUT...

### ISSUE THREAD

**Issue by nvisionary911** [kijai/ComfyUI-HunyuanVideoWrapper#150]
So I deleted comfy UI and all the folders to start a brand new instance and install, now the new comfyui is opening into the Hunyan wrapper workflow, obviously missing every single component? what do I do wrong? I though deleting the folders was all the "uninstall", my machine remembers it still. can anyone help me please? I wanted to go from scratch again since thats a solution for many people running Hunyan when it did not run properly at first

**Comment by Skol600ml** [comment]
It's the browser cache

**Comment by nvisionary911** [comment]
> It's the browser cache

thanks, will check it out, I'm a traditional film maker so this is all new to me, is there any way to set which browser to run Comfy? Because now default it opens up in one I don't actually use

**Comment by maplecasino** [comment]
Did you open the Manager and Install Missing Custom Nodes?

**Comment by zeldapkmn** [comment]
Yeah, update broke my workflow and keep getting OOM/Allocation errors now
